src/dotnet-scaffolding/Microsoft.DotNet.Scaffolding.Helpers/Services/CodeService.cs

```diff
@@ -126,7 +126,7 @@ public void Dispose()
         UnloadWorkspace();
     }
 
-    public async Task<IList<ISymbol>> GetAllClassSymbolsAsync()
+    public async Task<List<ISymbol>> GetAllClassSymbolsAsync()
```

Comment:
We'll want to API review this if this class (or a derivative of it) gets opened up as part of the public API. For now this is fine.